src/Symfony/Component/Security/Core/Authorization/DebugAccessDecisionManager.php

```diff
@@ -29,7 +29,7 @@ class DebugAccessDecisionManager implements AccessDecisionManagerInterface
     private $voters;
     private $decisionLog = array();
 
-    public function __construct(AccessDecisionManager $manager)
+    public function __construct(AccessDecisionManagerInterface $manager)
```

Comment:
:+1: I missed this PR, so in case it helps: https://github.com/symfony/symfony/compare/3.1...Ener-Getick:SECURITY